### PR TITLE
Document applications & teams

### PR DESCRIPTION
**Pull request by dolfies**

### CODE REVIEW

File: pages/resources/application.mdx [hunk 1]
```diff
@@ -4,153 +4,587 @@
...
+| eula_id?                                     | snowflake                                                               | The ID of the [EULA](/resources/store#eula-object) required to play the application's game                                                                                           |
+| slug? ^1^                                    | string                                                                  | The URL slug that links to the primary store page of the application                                                                                                                 |
+| aliases?                                     | array[string]                                                           | Other names the application's game is recognized by                                                                                                                                  |
```
Comment (splatterxl) [inline pages/resources/application.mdx:21]:
```suggestion
| aliases?                                     | array[string]                                                           | Other names the application's game is associated with                                                                                                                                  |
```

File: pages/resources/application.mdx [hunk 2]
```diff
@@ -160,6 +594,394 @@ Each metadata type offers a comparison operation that allows guilds to configure
...
+| custom_install_url?                      | ?string                                                                  | The default custom authorization link for the integration                                                                                                                                |
+| discoverability_state?                   | integer                                                                  | The current application directory [discoverability state](#application-discoverability-state) of the application (only `NOT_DISCOVERABLE` and `DISCOVERABLE` is supported)               |
+| max_participants?                        | ?integer                                                                 | The maximum number of people that can participate in the application's embedded activity (-1 for no limit)                                                                               |
```
Comment (splatterxl) [inline pages/resources/application.mdx:695]:
```suggestion
| max_participants?                        | ?integer                                                                 | The maximum possible participants in the application's embedded activity (-1 for no limit)                                                                               |
```

File: pages/resources/application.mdx [hunk 3]
```diff
@@ -4,153 +4,587 @@
...
+| interactions_version                         | integer                                                                 | The [version of the application's interactions endpoint implementation](#application-interatcions-version)                                                                           |
+| interactions_event_types ^4^                 | array[string]                                                           | The enabled [Gateway events](/topics/gateway-events) to send to the interaction endpoint                                                                                             |
+| tags?                                        | array[string]                                                           | Tags describing the content and functionality of the application (max 20 characters, max 5)                                                                                          |
```
Comment (splatterxl) [inline pages/resources/application.mdx:47]:
```suggestion
| tags?                                        | array[string]                                                           | Tags describing the content and functionality of the application, processed to categorise the app in the App Directory (max 20 characters, max 5)                                                                                          |
```

File: pages/resources/team.mdx [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+}
+```
+
+<RouteHeader method="GET" url="/teams/{team.id}/payouts/onboarding">
+  Get Team Payout Onboarding
+</RouteHeader>
+
+Returns a link that can be embedded in an IFrame to allow the user to access the team's Tipalti payout account dashboard. User must be the owner of the team.
+
+<Alert type="warn">
```
Comment (splatterxl) [inline pages/resources/team.mdx:349]:
```suggestion
<Alert type="info">
```

File: pages/topics/oauth2.mdx [hunk 5]
```diff
@@ -376,26 +376,26 @@ From this object, you should store the `webhook.token` and `webhook.id`. See the
...
 
-Returns the bot's [application](/resources/application#application-object) object without `flags`.
+Returns info about the current authorization.
 
-<RouteHeader method="GET" url="/oauth2/@me">
-  Get Current Authorization Information
-</RouteHeader>
+<Alert type="warn">
+
+This endpoint is not usable by user or bot accounts.
```
Comment (splatterxl) [inline pages/topics/oauth2.mdx:387]:
```suggestion
This endpoint is only usable through OAuth2.
```

**Review by splatterxl — CHANGES_REQUESTED** [review]

File: pages/resources/application.mdx (hunk 3 above)
Comment (dolfies) [reply pages/resources/application.mdx:48]: I don't think this is only for app directory

File: pages/reference.mdx [hunk 6]
```diff
@@ -407,7 +407,7 @@ In the case of endpoints that support animated images, the hash will begin with
...
 
-^1^ In the case of the Default User Avatar endpoint, the value for `user_index` in the path for migrated users should be the user's ID shifted 22 bits to the left modulo 6 (i.e. `user_id >> 22 % 6`). The value for non-migrated users the user's discriminator modulo 5 (Test#1337 would be `1337 % 5`, which evaluates to 2). See the [section on Discord's new username system](/resources/user#unique-usernames) for more information.
+^1^ In the case of the Default User Avatar endpoint, the value for `user_index` in the path for migrated users should be the user's ID shifted 22 bits to the left modulo 6 (i.e. `user_id >> 22 % 6`). The value for non-migrated users the user's discriminator modulo 5 (Test#1337 would be `1337 % 5`, which evaluates to 2). The value for teams should be the team's ID modulo 5 (i.e. `team_id % 5`). See the [section on Discord's new username system](/resources/user#unique-usernames) for more information.
```
[the remark this answers is not in the document]
Comment (dolfies) [reply pages/reference.mdx:410]: It's also explained in the user resource.

File: pages/topics/oauth2.mdx (hunk 5 above)
Comment (arHSM) [reply pages/topics/oauth2.mdx:387]: I agree with this suggestion

**Review by arHSM — DISMISSED** [review]

File: pages/resources/team.mdx (hunk 4 above)
Comment (dolfies) [reply pages/resources/team.mdx:349]: I want this to be a different color!!

**Review by splatterxl — APPROVED** [review]

**Review by arHSM — APPROVED** [review]

**Review by Dziurwa14 — APPROVED** [review]

**Pull request merged**